services/cincinnati-ci/README.md

```diff
@@ -1,8 +1,12 @@
-# Project declarations for the cincinnati-ci
+# Project declarations for the openshift/cincinnati CI
 
 This service declares resources used by the cincinnati CI jobs.
 
-This namespace and its resources are dedicated to the cincinnati CI jobs.
+These namespaces and their resources are dedicated to the cincinnati CI jobs.
 Initially we're interested in using it for secrets and and container images.
 jobs. Initially we're interested in using it for secrets and and container
 images.
+
+Namespace and their purposes:
+* `cincinnati-ci`: holds secrets and images which are accessible only to authorized users.
+* `cincinnati-ci-public`: holds images which are publicly accessible
```

Comment:
```suggestion
* `cincinnati-ci-public`: holds images which are publicly accessible. These images are used by cincinnati's unit tests, so these have to be pullable without a pull secret
```

Reply:
The unit tests could actually use authorization as well, but initially I restricted only a small subset of unit tests to make use of it because it was a new feature. The point of having both is to guarantee that both authorized and unauthorized access works.

As of now I'm undecided which test cases we should run through which namespace. We could duplicate all test cases to the public and private namespaces and assert equal results. OTOH, it should be possible to test authentication separately, and abstract the rest of the use-cases. In this case I'd probably default to making the images public, to simplify debugging. WDTY?

Reply:
I'd prefer most test images to be open, except for the part which deals with authentication (do we even have those?). Not sure if its worth duplicating both cases for auth and non-auth case: verifying all possible combinations doesn't really fit unit test goals.

e2e tests however may access authenticated images as CI install pull secret on the clusters - but we use production images for those.

Reply:
Sounds like we have an agreement on trying to minimize the private tests to a bare minimum to ensure that the authentication mechanism doesn't break. 

WRT your suggestion, it's somewhat a grey area to me if we call tests which access the internet "unit tests". The tests in the cincinnati repo which aren't feature gated with `test-net` or `test-net-private` don't rely on network resources, and they're the ones which I would regard as unit tests.

Reply:
>Sounds like we have an agreement on trying to minimize the private tests to a bare minimum to ensure that the authentication mechanism doesn't break.

:+1: 

>WRT your suggestion, it's somewhat a grey area to me if we call tests which access the internet "unit tests".

Oh, right, I assumed any test which is not e2e to be a unit test. I'd prefer tests which use networking to use public images as much as possible (so that non Red Hat users could easily run those). Its not critical though

Reply:
>  I assumed any test which is not e2e to be a unit test.

The distinction isn't really clear to me at this point because feature-gates allow to group tests further. I suggest we don't specify it in this README

Reply:
We do need to set some generic information for CI engineers so that they'd have some idea about why two namespaces are required.

Lets settle on simply "network tests"?

Reply:
Well, both are used for network tests ;-) I added some distinguishing words. PTAL